### PR TITLE
Basic structure

### PR DESCRIPTION
**Pull request by Laylo309**
- Create an npm project with webpack. 
- Use ES6 modules, with import and export.
- Write plain HTML markup with minimum styling.
- No linter errors 😎

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+  module: {
```
Comment (rloterh) [inline webpack.config.js:22]: - [x] Your webpack configuration is missing the `webpack dev-server`. Kindly ensure to fix this in your webpack configuration to enable the application to be run on a server

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Laylo, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦🚦

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check these comments, and comments below this review_
- [x] Your project structure is lacking the `\dist` directory. Kindly ensure to include or render this directory, together with the `.js` and `.html` files

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack.config.js (hunk 1 above)
Comment (henatan99) [reply webpack.config.js:22]: This issue is resolved well! Excellent work! :champagne:

**Review by henatan99 — APPROVED** [review]
Hi @Laylo309 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- For the dist folder mentioned by the previous reviewer I recommend you not to include it in `.gitignore` for your Microverse projects. :+1: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**